website/docs/cli/commands/version.mdx

```diff
@@ -20,7 +20,19 @@ Execute the `atmos version` command like this:
 atmos version
 ```
 
-This will show the CLI version.
+This will show the CLI version. 
+The frequency with which the update box message in version is shown is configured in atmos.yaml.
+There are two options daily and weekly. The default value is set to daily. 
+It is also possible to turn it off in atmos.yaml file by switching enable to false. 
+Or set an env variable ATMOS_VERSION_CHECK_ENABLED to false, 
+this overriders the config's settting. 
```

Comment:
```suggestion
This will show the CLI version. 

From time to time, Atmos will check for updates. The frequency is configured in the `atmos.yaml`. There are two options `daily` and `weekly`. The default is to check `daily`, and if any unsupported values are passed this default will be used.

It is also possible to turn off version checks in `atmos.yaml` by switching `enable` to `false`, or by setting the `ATMOS_VERSION_CHECK_ENABLED` environment variable to `false`, which overriders the config's settings. 
```